src/Mod/Fem/Gui/ViewProviderFemPostObject.cpp

```diff
@@ -1019,8 +1023,13 @@ bool ViewProviderFemPostObject::setEdit(int ModNum)
 void ViewProviderFemPostObject::setupTaskDialog(TaskDlgPost* dlg)
 {
     assert(dlg->getView() == this);
-    auto panel = new TaskPostDisplay(this);
-    dlg->addTaskBox(panel->windowIcon().pixmap(32), panel);
+    auto disp_panel = new TaskPostDisplay(this);
```

Comment:
One thing is FreeCAD mostly uses `camelCase` for local variables - I think that it is quite important to keep that consistent across codebase.

Reply:
I agree.